server/datastore/mysql/migrations/tables/20231002120317_AddDiscardToQueries_test.go

```diff
@@ -0,0 +1,37 @@
+package tables
+
+import (
+	"testing"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUp_20231002120317(t *testing.T) {
+	db := applyUpToPrev(t)
+	applyNext(t, db)
```

Comment:
lets add a test showing that we get the default value on existing rows when adding the new column

Reply:
done!